react/week2/todolist/src/App.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import "./App.css";
+import TodoList from "./components/TodoList";
+import Timer from "./components/Timer";
+import AddTodo from "./components/AddTodo";
+
+const listOfTodos = [
+  { id: 1, description: "Get out of bed" },
+  { id: 2, description: "Brush teeth" },
+  { id: 3, description: "Eat breakfast" },
+];
+
+function App() {
+  const [todos, setTodo] = useState(listOfTodos);
+  let newTodos = [...todos];
+
+  const addTodo = () => {
+    const newTodo = { id: todos.length + 1, description: "Random text" };
```

Comment:
As a new todo id, try to use something that is unique. Imagine you remove a todo from the middle of the list, then the new todo you're going to add will have the same id as the last todo added!
It is similar to using an index as a key, have a look at this link which also suggest alternatives : https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Thank you I've changed the id to be unique.  👍